s4e-web/src/app/views/map-view/user-dropdown/user-dropdown.component.html

```diff
@@ -63,7 +63,7 @@
           target="_blank"
           i18n
         >
-          Zapisz źródło widoku (surowe dane)
+          Metadane i dane surowe
```

Comment:
Dane surowe i metadane
może brzmieć lepiej?

Reply:
Missed that. Sorry. The original wording reflects the order in the modal that pops up